### ISSUE TITLE
Question: How do I turn off rm -i ?

### ISSUE THREAD

**Issue by Industrial** [sorin-ionescu/prezto#1377]
How do I set rm to never ask for confirmation?

I think it's a counterproductive way of using a computer and it belongs with the (l)users in GUI-land where people thank their computer for saving their ass instead of not making stupid mistakes.

**Comment by belak** [comment]
Hi @Industrial, thanks for asking. We actually just resolved #205 a few days ago to provide an option to disable these because people asked to have them disabled completely. It's documented in the zpreztorc and the README for the utility module.

https://github.com/sorin-ionescu/prezto/blob/master/runcoms/zpreztorc#L194-L197

https://github.com/sorin-ionescu/prezto/tree/master/modules/utility#safe-ops

Anyway, long story short, they can be disabled by adding `zstyle ':prezto:module:utility' safe-ops no` to your zpreztorc (or to your zshrc before prezto is loaded, depending on your setup).

If you have any suggestions for making those more visible or easier to search for, I'd be happy to hear them. Hope this helps!

**Comment by Industrial** [comment]
Thanks for the quick reply!

**Comment by belak** [comment]
Good timing, I guess. :)

I'm going to close this for now because there's a way of disabling them, but feel free to leave any comments/suggestions you have (or if you're having trouble getting it working) and I'll reopen the issue if needed.

Cheers!
